[2944][2682][ADD] auction_seller

AungKoKoLin1997 opened the pull request:
[2944](https://www.quartile.co/web#id=2944&model=project.task&view_type=form&menu_id=)
I got the latest status from 2683-add-auction_seller branch from rmm-private
Recreated for this [PR](https://github.com/qrtl/rmm-private/pull/39)
[2946](https://www.quartile.co/web#id=2946&action=771&model=project.task&view_type=form&menu_id=505)
[2682](https://www.quartile.co/web?#id=2682&action=771&model=project.task&view_type=form&menu_id=505)

AungKoKoLin1997 committed:
[ADD] auction_seller

AungKoKoLin1997 committed:
add event dependency

yostashiro commented:
@AungKoKoLin1997 Diff of this PR is +1165 while that of the other one [#39](https://github.com/qrtl/rmm-private/pull/39) is +1155.  Where does the difference of 10 lines come from?

AungKoKoLin1997 commented:
> @AungKoKoLin1997 Diff of this PR is +1165 while that of the other one [#39](https://github.com/qrtl/rmm-private/pull/39) is +1155. Where does the difference of 10 lines come from?

@yostashiro We changed the module license from Other proprietary to LGPL-3. So, the lincense line will be added in readme, index.html, etc. And I also added license line in .py file.

yostashiro commented:
@AungKoKoLin1997 Got it. Missed that because the change was already included in your initial commit.

yostashiro commented on auction_seller/models/seller_board.py:
```diff
@@ -0,0 +1,13 @@
+    _name = "seller.board"
+
+    name = fields.Char()
+    state = fields.Selection(
```
I think this field name `state` is misleading.  How did we pick this field name?  I think something like `board_type` would be better.

kanda999 replied:
I used sst V11 dashboard as a reference.   `state` was used on that dashboard.

yostashiro commented on auction_seller/models/seller_board.py:
```diff
@@ -0,0 +1,13 @@
+
+    name = fields.Char()
+    state = fields.Selection(
+        [("product", "Product"), ("item", "Item"), ("order", "Order")]
```
```suggestion
        [("product", "Products"), ("sold_item", "Sold Items"), ("order", "Orders")]
```

kanda999 commented:
@AungKoKoLin1997 
Could you follow up on this ?
If field name chenge, you need to adjust https://github.com/qrtl/rmm-custom/pull/42

AungKoKoLin1997 commented:
> Could you follow up on this ?
> If field name chenge, you need to adjust #42

Yes. I need to adjust https://github.com/qrtl/rmm-custom/pull/42. Will you take care this PR or me?

kanda999 commented:
@AungKoKoLin1997 
Please follow it...! #42

AungKoKoLin1997 committed:
[FIX] change field name and adjust selection field

AungKoKoLin1997 commented:
We need to adjust PRs https://github.com/qrtl/rmm-private/pull/40 and https://github.com/qrtl/rmm-custom/pull/42 to compatible with the latest commit. In latest commit, remove some selection items because this item should be added in their own module.
